examples/src/main/scala/org/apache/spark/examples/ml/VectorIndexerExample.scala

```diff
@@ -41,8 +41,8 @@ object VectorIndexerExample {
     val indexerModel = indexer.fit(data)
 
     val categoricalFeatures: Set[Int] = indexerModel.categoryMaps.keys.toSet
-    println(s"Chose ${categoricalFeatures.size} categorical features: " +
-      categoricalFeatures.mkString(", "))
+    println(s"Chose ${categoricalFeatures.size} " +
+      s"categorical features: {$categoricalFeatures.mkString(", ")}")
```

Comment:
Typo here. Please make sure build/test/style passes locally before pushing again, or else this takes a lot of work to review.

Reply:
I did fixed this. Can you please give me steps as a check list before commit for test.

Reply:
`./dev/run-tests`